assets/express/src/api/user/dto/user.dto.ts

```diff
@@ -0,0 +1,38 @@
+import { z } from 'zod';
+import { User } from '@entities';
+
+export const createUserRequestPayload = User.omit({ role: true });
+
+export const loginUserRequestPayload = z.object({
+  email: z.string(),
+  password: z.string({
+    required_error: 'Password is required',
+  }),
+});
+
+export const updateUserRequestPayload = User;
```

Comment:
Instead of `RequestPayload` we can use `DTO` since `DTO` means Data Transfer Object or just nothing since this lives in the DTO folder.